ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpService.java

```diff
@@ -320,6 +321,7 @@ private Router buildRouter() {
         .route()
         .handler(
             BodyHandler.create()
+                .setBodyLimit(BODY_LIMIT)
```

Comment:
If this limit is breached, what error is returned? And do we translate that to a JsonRpcError?

Reply:
> If this limit is breached, what error is returned? And do we translate that to a JsonRpcError?

HTTP response status 413 - `Request Entity Too Large`. It does not translate to a JsonRpcError since it is more like a client error.